data_collection/gazette/spiders/base/nucleogov.py

```diff
@@ -0,0 +1,41 @@
+import json
+from datetime import datetime
+
+import scrapy
+from dateutil.rrule import DAILY, rrule
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class NucleoGovGazetteSpider(BaseGazetteSpider):
+    def start_requests(self):
+        days = rrule(freq=DAILY, dtstart=self.start_date, until=self.end_date)
+        for day in days:
+            yield scrapy.Request(self.url_base.format(day.strftime("%Y-%m-%d")))
```

Comment:
como a string de acesso a API `/api/diarios?data={}&calendar=true&situacao=2"` é igual em todas as classes filhas, nós temos adotado o padrão de deixar essa construção da URL na classe mãe, escondendo isso da classe filha e deixando mais simples o raspador do município.

```suggestion
            url = f"{self.url_base}/api/diarios?data={day.strftime('%Y-%m-%d')}&calendar=true&situacao=2"
            yield scrapy.Request(url)
```